web/packages/teleterm/src/ui/services/workspacesService/documentsService/connectToApp.ts

```diff
@@ -46,6 +50,23 @@ export async function connectToApp(
     return;
   }
 
+  if (isWebApp(target) && options?.launchInBrowserIfWebApp) {
+    captureAppLaunchInBrowser(ctx, target, telemetry);
+    const rootCluster = ctx.clustersService.findCluster(
+      routing.ensureRootClusterUri(target.uri)
+    );
+    const cluster = ctx.clustersService.findClusterByResource(target.uri);
+    // Generally, links should be opened with <a> elements.
+    // Unfortunately, in some cases it is not possible,
+    // for example, in the search bar.
+    window.open(
+      getWebAppLaunchUrl({ app: target, rootCluster, cluster }),
+      '_blank',
+      'noreferrer'
```

Comment:
Should we have noopener here too? It's generally recommended for any links that open _blank, and I don't think the opened app needs to know `window.opener`

Reply:
Added.

Reply:
Now that I think about it, I'm not sure if noreferrer and noopener have any effect here. Those attributes are used in browsers when navigating from one page to another. But here the Electron app is going to open the page in an external browser. The only thing it passes forward is the URL. Still a good practice though.